service/src/main/java/bio/terra/workspace/common/utils/StepResultWithFlightInfo.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.workspace.common.utils;
+
+import bio.terra.stairway.*;
+import javax.annotation.Nullable;
+
+public class StepResultWithFlightInfo {
```

Comment:
I didn't write a unittest for this class because it is mostly just pass-throughs, but I can add one if folks think it's worthwhile.